scalatest/src/main/scala/org/scalatest/enablers/InspectorAsserting.scala

```diff
@@ -324,6 +326,264 @@ abstract class UnitInspectorAsserting {
     }
 }
 
+trait FutureInspectorAsserting {
```

Comment:
@cheeseng any reason this is a separate trait? The reason UnitInspectorAsserting is in a separate trait is to make it lowest priority implicit. If not, I'd prefer we eliminate this trait and just define this Impl class in the object and make it private to minimize API surface area.